highfidelity/hifi
# Fix Interstitial GoTo button stretch

- Leave interstitial page GoTo button in its natural dimensions

## TEST PLAN
- Launch Interface
- - Enable `Settings > Developer Menu`
- Open `Developer > Console`
- Enter in `Settings.setValue("enableInterstitialMode", true);`
- Relaunch Interface 
- When navigating to a domain, the GOTO button should look like the one below (not stretched)
![image](https://user-images.githubusercontent.com/10539521/45842979-22a28e00-bcd3-11e8-9100-ffe157f2518c.png)
- Try this both in Desktop and HMD mode
